upload/views/bundle_analysis.py

```diff
@@ -73,6 +76,9 @@ def post(self, request):
         else:
             raise NotAuthenticated()
 
+        if repo is None:
```

Comment:
The authentication classes don't check if the repo exists already?
(asking for my own understanding)

Reply:
I don't believe so, I just added this to appease the type checker because if the repo doesn't exist it will just report null for the repo var.